frontend/src/components/About.vue

```diff
@@ -6,48 +6,139 @@
+          The development of Metabolic Atlas is led by
+          <a href="https://nbis.se/about/staff/mihail-anton" target="_blank">Mihail Anton</a>, with
+          team members Pierre-Etienne Cholley and Shan Huang. Past and more detailed contributions
+          are
+          <a
+            href="https://github.com/SysBioChalmers/MetabolicAtlas/graphs/contributors"
+            target="_blank"
+            >shown on GitHub</a
+          >. For versions and release notes of the software behind Metabolic Atlas,
+          <a href="https://github.com/SysBioChalmers/MetabolicAtlas/releases" target="_blank"
+            >check the GitHub page</a
+          >
+          or
+          <a href="https://zenodo.org/badge/latestdoi/53664497" target="_blank">Zenodo</a>. The
+          integrated models are also available on GitHub, check our
+          <router-link :to="{ name: 'gems' }">GEM Repository</router-link> for links to those.<br /><br />
 
-          <p>Metabolic Atlas would not be possible without the following excellent Elixir resources: <a href="https://proteinatlas.org" target="_blank">The Human Protein Atlas</a>, <a href="https://www.ensembl.org/" target="_blank">Ensembl</a>, <a href="https://www.uniprot.org" target="_blank">UniProt</a>, <a href="https://europepmc.org" target="_blank">Europe PMC</a>, <a href="https://identifiers.org" target="_blank">Identifiers.org</a>, <a href="https://www.ebi.ac.uk/chebi" target="_blank">ChEBi</a> and other essential sources of information: <a href="http://www.genome.jp/kegg/" target="_blank">KEGG: Kyoto Encyclopedia of Genes and Genomes</a>,  <a href="http://www.hmdb.ca/" target="_blank">The Human Metabolome Database</a>, <a href="https://www.metanetx.org" target="_blank">MetaNetX</a>, <a href="http://bigg.ucsd.edu" target="_blank">BiGG</a>, <a href="https://www.ncbi.nlm.nih.gov" target="_blank">NCBI</a>.
+          <p>
+            Metabolic Atlas would not be possible without the following excellent Elixir resources:
+            <a href="https://proteinatlas.org" target="_blank">The Human Protein Atlas</a>,
+            <a href="https://www.ensembl.org/" target="_blank">Ensembl</a>,
+            <a href="https://www.uniprot.org" target="_blank">UniProt</a>,
+            <a href="https://europepmc.org" target="_blank">Europe PMC</a>,
+            <a href="https://identifiers.org" target="_blank">Identifiers.org</a>,
+            <a href="https://www.ebi.ac.uk/chebi" target="_blank">ChEBi</a> and other essential
+            sources of information:
+            <a href="http://www.genome.jp/kegg/" target="_blank"
+              >KEGG: Kyoto Encyclopedia of Genes and Genomes</a
+            >, <a href="http://www.hmdb.ca/" target="_blank">The Human Metabolome Database</a>,
```

Comment:
It looks weird here as well for `<a>`.

Reply:
Fixed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/797/commits/a21f080b80e8b59f661b064710407d0b5b947a34#diff-347c85641aa3243ab418a0dfededf202763a1d5a28dc34eacb00b912b1a97fe4R5